### ISSUE TITLE
Daily limit by custom time [feature-request]

### ISSUE THREAD

**Issue by allencch** [rcaelers/workrave#93]
Currently daily limit is timer. It would be good if it can prompt the user to shutdown at custom time set by user.

**Comment by easpro** [comment]
I agree.  A user should be able to set a start time and end time for each day of the week.  At the end of set time, an option to lock screen, log off or shutdown.

**Comment by Chealer** [comment]
Depending on what you mean by "shutdown", this probably requests the same as ticket #514, which has a clearer title.

**Comment by allencch** [comment]
Shutdown means turn off the computer. Agree it is the same as the ticket #514 . Closing this.
